### PR TITLE
Update tbcQuestFixes.lua

### PR DESCRIPTION
**Pull request by GillsITWorld**
Add Extra Objectives for Quest 11885 (Fix 2 of 2)
Tested on Beta

### CODE REVIEW

**Comment by BreakBB** [comment]
Hey @GillsITWorld thanks for taking your time and digging into the Questie corrections!

Let me explain two things, which will hopefully help you with this and future PRs for quest corrections :)

1) The `extraObjective` field can also take database references, so you don't have to copy over spawns of existing NPCs/objects. But you couldn't know this since the explanation of the `extraObjective` field is not complete (yet), sorry 🙈 You would add a database reference like this:

```lua
        [8490] = {
            [questKeys.extraObjectives] = {{nil, ICON_TYPE_EVENT, "Place the Infused Crystal and protect it from the Scourge for 1 minute", 0, {{"object", 181164}}}}
        },
```

After the `0` for an optional objective index, you open another table which contains pairs of the database type you want to reference (npc, object, item) and the ID of the reference. The example above references the object 181164.

2) We actually don't want to show the "spawns" of NPCs which spawn you have to trigger in some way. Instead mark the location where you have to summon something/someone. This could be done either with an `ICON_TYPE_OBJECT` if you have to interact with some game object or `ICON_TYPE_EVENT` if you have to use a quest item.

Also in most cases you don't need the indivdual spawns at the beginning of the `extraObjectives` field, but can reference an object where something has to happen. 

For the quest you are trying to fix [Adversarial Blood](https://www.wowhead.com/quest=11885/adversarial-blood) you want to search for the "Skull Pile" object with which you can summon the NPCs you have to kill.

Hit me up on Discord or simply ask here if you have more questions. I am always happy to get help fixing all the quest bugs 😁

**Comment by GillsITWorld** [comment]
Ok, I understand this, so how do we label/icon who you farm to get the Time-Lost Scrolls from to be able to use the Skull Piles?

**Comment by GillsITWorld** [comment]
[https://i.imgur.com/YxRKN17.png](url)

https://imgur.com/YxRKN17

**Comment by BreakBB** [comment]
Items you need to complete the objectives, but which are not the objectives themself can be added with the `requiredSourceItem` field. As long as the item is linked to NPCs those are shown on the map just fine.

Also above you have a "Red" in your color code which shouldn't be there. And I am not really sure what to say about the formatting of the text you did. It is a great idea, but there should be a standard way to do this 🤔 For now it would be better to remove the formatting as it should not be directly included in the database imo.

**Comment by GillsITWorld** [comment]
So remove the color codes, leave basic text?

**Comment by BreakBB** [comment]
For now that would be perfect yes.

Optionally create an issue with your suggestion towards the formatting, since that makes it even easier to understand what to do/use when and where and how :)

File: Database/Corrections/TBC/tbcQuestFixes.lua [hunk 1]
```diff
@@ -1103,6 +1103,9 @@ function QuestieTBCQuestFixes:Load()
...
...},{61.4,75.4},{61.4,75.6},{61.4,78},{61.6,78},{61.8,75.6},{62,75},{62.2,74.4},{65.8,79.4},{66,79.6},{6.8,80.2},{68.4,85.2},{68.6,74.4},{69,85.6},{69.2,85.4},{69.4,74.8},{69.4,75.6},{69.6,74.4},{69.6,74.8},{69.6,76},{69.6,84.8},{69.8,79.6},{70,79.4},{70,83.6},{70.2,83.4},{70.4,82},{70.6,82.2},{71.8,85.6},{72,85.4},{72.2,88},{72.8,87.8},{73,84.4},{73,84.8},{73.4,82},{73.6,87.8},{73.8,83.2},{73.8,86},{73.8,86.6},{74.4,81},{74.4,84.2},{74.6,83},{74.6,87.4},{74.6,87.6},{75.2,81.2},{75.2,81.6},{60.8,75.6},{61,74.8},{61,80.2},{61.2,79.2},{61.4,73.6},{61.8,73.4},{61.8,73.6},{62,74.6},{63,79.6},{65.8,78.2},{65.8,79.2},{66.4,79.8},{67.2,79},{67.4,79.8},{67.6,79.8},{68.2,73.8},{68.8,74},{69,78.8},{69.2,78.4},{69.4,74.6},{69.6,78.4},{69.8,75.2},{70,79.4},{70,79.6},{70.2,74.4},{70.2,83.2},{70.4,85},{70.6,84.2},{70.6,85},{71.2,82.4},{71.2,82.6},{71.6,81.8},{72,85.4},{72,85.6},{72.4,88},{73,82.8},{73,83.6},{73.4,80.2},{73.4,80.6},{73.6,81},{74,80.2},{74.4,88.4},{74.6,88.2},{74.6,88.6},{75,81.2},{75.2,86},{75.2,86.8},}}, ICON_TYPE_LOOT, "Kill |cFFFF0000Red Time-Lost Skettis|r for |cFFFFFFFF[Time-Lost Scrolls]|r, must have |cFFFFFFFF[Elixir of Shadow]|r consumed from previous repeatable quest", 0}}
```
Comment (BreakBB) [inline Database/Corrections/TBC/tbcQuestFixes.lua:1107]: Remove the spawnlist, add a dbReference and remove the text formatting (for now) :)

**Review by BreakBB — CHANGES_REQUESTED** [review]

**Comment by BreakBB** [comment]
Closing this in favor of #2690

**Pull request closed without merging**